library/bignum_mod_raw.c

```diff
@@ -127,7 +127,40 @@ int mbedtls_mpi_mod_raw_write( const mbedtls_mpi_uint *A,
 /* END MERGE SLOT 6 */
 
 /* BEGIN MERGE SLOT 7 */
+int mbedtls_mpi_mod_raw_to_mont_rep( mbedtls_mpi_uint *X,
+                                     const mbedtls_mpi_mod_modulus *m )
+{
+    const mbedtls_mpi_uint one = 1;
+    const size_t t_sz = ( m->limbs * 2 ) + 1;
+    mbedtls_mpi_uint *t;
+
+    if( ( t = (mbedtls_mpi_uint*)mbedtls_calloc( t_sz, ciL ) ) == NULL )
+        return( MBEDTLS_ERR_MPI_ALLOC_FAILED );
```

Comment:
(for discussion, for later) is there a way we could look at using a stack-based variable-length array, as we require c99? this would be a lot faster. Might not be suitable on small systems with large MPIs, but for EC could be useful